inc/player/p_index.hpp

```diff
@@ -25,7 +29,41 @@ class PIndex
     /// restores the content of the class from the zip string
     void Restore(std::string zipped);
 
```

Comment:
The class should be able to return a specific album. Something like
PAlbum getAlbum(std::string album_path)
where album_path is specific enough for Index class to figure out which album it should return.